### PR TITLE
Profiling corrected.

### PR DESCRIPTION
**Pull request by josephjohnjj**
In Parsec Trace Table format V2 the information was stored incorrectly.

### CODE REVIEW

**Comment by omor1** [comment]
`tools/profiling/dbpreader.c` seems to have been committed extraneously.
To clarify, this fixes a bug introduced by #432 that causes the PTT v2 format to be saved incorrectly if only one thread is used; the `event_infos` dictionary was being dropped.

File: tools/profiling/dbpreader.c [hunk 1]
```diff
@@ -780,7 +780,8 @@ int dbp_reader_last_error(const dbp_multifile_reader_t *dbp)
 
 void dbp_reader_close_files(dbp_multifile_reader_t *dbp)
 {
-    for( int i = 0; i < dbp->nb_files; i++ ) {
```
Comment (therault) [inline tools/profiling/dbpreader.c:783]: Indentation of `int i = 0;` should be aligned with the `for`.

**Review by therault — APPROVED** [review]
Please fix indentation before merging. Otherwise LGTM.

File: tools/profiling/dbpreader.c (hunk 1 above)
Comment (bosilca) [reply tools/profiling/dbpreader.c:783]: Do not rearrange other people's code. Remove this please.

Comment (josephjohnjj) [reply tools/profiling/dbpreader.c]: Sorry, that was accidental.

**Review by bosilca — APPROVED** [review]

**Pull request merged**